src/python/Makefile

```diff
@@ -0,0 +1,88 @@
+.PHONY: clean clean-test clean-pyc clean-build docs help
```

Comment:
This file adds implicit dependencies on several other tools, including `flake8`, `sphinx-adoc`, `pytest`, `coverage`, and `tox`.

If we want to use this Makefile, we should either add these tools as project dependencies, or provide a setup script just for this cli package to install these.